HW1/P5/P5.txt

```diff
@@ -1 +1,4 @@
 # Your discussion here
+
+['Harvard_University','Barack_Obama','Kevin_Bacon']
+[‘Kevin_Bacon’,’Barack_Obama’,’Harvard_University’]
```

Comment:
I did not find these paths :( ... and my distance between Kevin Bacon to Harvard University was 2 and the other way round was 3. I will check both of our codes to see if one of us is wrong.